scripts/modules/cli.py

```diff
@@ -606,7 +607,7 @@ def start_handler(self):
                 del services["opbeans-load-generator"]
 
         compose = dict(
-            version="2.1",
+            version="2.4",
```

Comment:
I'm not sure if this will cause an issue with the `"condition": "service_healthy"`, as far as I see 2.4 is supported -> https://docs.docker.com/compose/compose-file/compose-file-v2/#depends_on So i said nothing

Reply:
no, this is on 3.0